### DISCUSSION TITLE
Metrics comparing docTR to other available OCR's

### DISCUSSION THREAD

**Discussion by aditya-ihx** [mindee/doctr#1576] [Ideas]
There is no official metric to show how much we better/worse we perform comparing to other OCR's like pytesseract or Easy-OCR. How about having a metric section for the same. This can be helpful so that others can find docTR more as well as build trust among the users  who are skeptical

**Comment by felixdittrich92** [comment]
Hi @aditya-ihx :wave: 

We have a metric comparision (with the cloud provider OCR solutions) :)
https://mindee.github.io/doctr/using_doctr/using_models.html#end-to-end-ocr

**Reply by felixdittrich92** [reply]
To compare with EasyOCR should be easy but about Tesseract i'm not sure because if i remember correctly it recognizes also whitespaces so it's not given that every prediction is on word boundary right ?

**Reply by SkaarFacee** [reply]
Yes that is true, I do feel some sort of metric would be helpful and beneficial

**Reply by SkaarFacee** [reply]
@felixdittrich92 We could also add in paddleOCR, as in most cases I have noticed EasyOCR and PaddleOCR to be the most common open source choice for an open source OCR

**Reply by SkaarFacee** [reply]
Hey, can anyone give me an update about this ?

**Reply by felixdittrich92** [reply]
On it :)
About the inference latency i have already added some benchmarks in https://github.com/felixdittrich92/OnnxTR

**Reply by SkaarFacee** [reply]
Oh yes, that looks perfect. Something similar for recall and precision would be perfect 🥳

**Reply by gkartik13** [reply]
Thanks for the latency comparison! But I was wondering if there are any plans on adding information comparing DocTR to EasyOCR, PaddleOCR and Tesseract in terms of precision  and recall?

**Reply by felixdittrich92** [reply]
Hey @gkartik13 :wave:,

Currently we have benchmarks against the big cloud provider OCR services (Google, AWS, Azure (Microsoft)): https://mindee.github.io/doctr/using_doctr/using_models.html#end-to-end-ocr

For EasyOCR this would be easy ^^ because it returns the results on word boundaries for PaddleOCR, pytesseract (subline / possible word pair results) / Surya (line boundaries) that's not the case - this would require some additional logic.

So everyone is invited to work on this - we are happy about every contribution - fell free to open an issue for this - but on our end some other topics have a much higher priority atm :)
